packages/cashier-v2/src/lib/WithdrawalCrypto/components/WithdrawalCryptoForm/WithdrawalCryptoForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Field, FieldProps, Formik } from 'formik';
+import { Button, Input } from '@deriv-com/ui';
+import { useWithdrawalCryptoContext } from '../../provider';
+import { validateCryptoAddress } from '../../utils';
+import { WithdrawalCryptoAmountConverter } from './components/WithdrawalCryptoAmountConverter';
+import { WithdrawalCryptoPercentageSelector } from './components/WithdrawalCryptoPercentageSelector';
+import styles from './WithdrawalCryptoForm.module.scss';
+
+const WithdrawalCryptoForm: React.FC = () => {
+    const { activeAccount, fractionalDigits, requestCryptoWithdrawal } = useWithdrawalCryptoContext();
+
+    return (
+        <Formik
+            initialValues={{
+                cryptoAddress: '',
+                cryptoAmount: '',
+                fiatAmount: '',
+            }}
+            onSubmit={values =>
+                requestCryptoWithdrawal({
+                    address: values.cryptoAddress,
+                    amount: parseFloat(parseFloat(values.cryptoAmount).toFixed(fractionalDigits.crypto)),
+                })
+            }
+        >
+            {({ errors, handleSubmit, isSubmitting, values }) => {
```

Comment:
could we extract:
const isSubmitButtonDisabled = Object.keys(errors).length !== 0 || !values.cryptoAmount || isSubmitting
and pass it to `disabled` prop?